File: FrontEnd/aseguradora/src/app/Models/Asegurado.ts
export class Asegurado {
    cedula:string;
    nombre_cliente: string;
    fecha_creacion: Date;
    telefono: string;
    edad: number;
  constructor() {
    this.cedula = '';
    this.nombre_cliente = '';
    this.fecha_creacion= new Date();
    this.telefono = '';
    this.edad = 0;
  }
  }

File: BackEnd/Consultorio_Seguros_Chubb/Models/Asegurado.cs
namespace Consultorio_Seguros_Chubb.Models
{
    public class Asegurado
    {
        public int AseguradoId { get; set; }
        public string cedula { get; set; }
        public string nombre_cliente { get; set; }
        public DateTime fecha_creacion { get; set; }
        public string telefono { get; set; }
        public int edad { get; set; }
        public bool estado { get; set; }
    }
}
